2022-11-13 21:38:58.135 | INFO     | __main__:main:91 - GuessNumberGame Server Start Successfully.
2022-11-13 21:39:03.927 | DEBUG    | __main__:route_ngg:22 - conn:<websockets.legacy.server.WebSocketServerProtocol object at 0x7fc38a063820>
2022-11-13 21:39:04.816 | INFO     | __main__:route_ngg:32 - got a message:{"api":"list_open_room", "data":{"username":"zxz_test1","roomname":"æˆ‘1"}}
2022-11-13 21:39:04.816 | INFO     | repository.crud:selectListByStatus:102 - [RoomCRUD::selectListByStatus]:sql=select * from room where status in (0, 1)
2022-11-13 21:39:04.848 | ERROR    | __main__:route_ngg:43 - [main.route_ngg] remote_address('101.64.140.35', 51063) errpr=Traceback (most recent call last):
  File "/root/code/pythoncode/PYProject/GuessNumberGame/src/main.py", line 39, in route_ngg
    send_msg = json.dumps(res)
  File "/root/miniconda3/envs/python39/lib/python3.9/json/__init__.py", line 231, in dumps
    return _default_encoder.encode(obj)
  File "/root/miniconda3/envs/python39/lib/python3.9/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/root/miniconda3/envs/python39/lib/python3.9/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "/root/miniconda3/envs/python39/lib/python3.9/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type bytes is not JSON serializable

